changelogs/unreleased/1337-skriss

```diff
@@ -0,0 +1 @@
+velero backup logs & velero restore logs: show helpful error message if item does not exist or is not finished processing
```

Comment:
Might want to clarify the text here a little bit - when I first read the title, I figured we'd be showing nicer errors when the logs file wasn't found.